packages/wpcom-checkout/src/transformations.ts

```diff
@@ -87,6 +87,37 @@ export function getTaxLineItemFromCart( responseCart: ResponseCart ): LineItem |
 	};
 }
 
+export function getTaxBreakdownLineItemsFromCart( responseCart: ResponseCart ): LineItem[] {
+	if ( ! responseCart.tax.display_taxes ) {
+		return [];
+	}
+	if (
+		! Array.isArray( responseCart.total_tax_breakdown ) ||
+		responseCart.total_tax_breakdown.length === 0
+	) {
+		const lineItem = getTaxLineItemFromCart( responseCart );
+		return lineItem ? [ lineItem ] : [];
+	}
+	return responseCart.total_tax_breakdown.map(
+		( taxBreakdownItem: TaxBreakdownItem, index: number ): LineItem => {
+			const id = `tax-line-item-${ index }`;
+			const label = taxBreakdownItem.label
+				? `${ taxBreakdownItem.label } (${ taxBreakdownItem.rate_display })`
```

Comment:
I need some feedback about whether this label needs to be localized.

Reply:
These seem like things that do not need to be localized here; I think that the `label` should be localized on the backend (if they even need localization) and `rate_display` is mostly numeric so it should be fine.